File: android/app/src/main/java/com/collectrndemo/modules/show/field/number/CardNumberModule.java
package com.collectrndemo.modules.show.field.number;

import androidx.annotation.NonNull;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;

import java.util.UUID;

public class CardNumberModule extends ReactContextBaseJavaModule {

    public CardNumberModule(ReactApplicationContext reactContext) {
        super(reactContext);
    }

    @NonNull
    @Override
    public String getName() {
        return "Show " + this.getClass().getSimpleName();
    }
}

File: android/app/src/main/kotlin/com/collectrndemo/simple/modules/collect/field/number/CardNumberModule.kt
package com.collectrndemo.simple.modules.collect.field.number;

import androidx.annotation.NonNull;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;

public class CardNumberModule extends ReactContextBaseJavaModule {

    public CardNumberModule(ReactApplicationContext reactContext) {
        super(reactContext);
    }

    @NonNull
    @Override
    public String getName() {
        return "Collect " + this.getClass().getSimpleName();
    }
}